pants

```diff
@@ -283,10 +283,12 @@ function scrub_env_vars {
   # Ensure the virtualenv PEX runs as shrink-wrapped.
   # See: https://github.com/pantsbuild/setup/issues/105
   local -r pex_env_vars=(${!PEX_@})
-  local -r pex_env_vars_to_scrub="${pex_env_vars[@]/PEX_ROOT}"
-  if [[ -n "${pex_env_vars_to_scrub[@]}" ]]; then
-    warn "Scrubbing ${pex_env_vars_to_scrub[@]}"
-    unset ${pex_env_vars_to_scrub[@]}
+  if [[ ! ${#pex_env_vars[@]} -eq 0 ]]; then
```

Comment:
What happened to the check I experimentally checked? Bash makes the sort of check you're doing here easier too with something like `if (( ${#pex_env_vars[@]} > 0 )); then`, but the single array declaration seems better. This extra check begs the question of why are we jumping though this extra check hoop?

Reply:
I provided an explanation here: https://github.com/pantsbuild/setup/pull/127#issuecomment-1232125314

Reply:
Alright, sounds good enough.